### ISSUE TITLE
roi integration

### ISSUE THREAD

**Issue by kklmn** [silx-kit/silx#3627]
Hi!

This was asked long time ago: #2627. Can you please tell the present status?

Let me rephrase the question. Given a roiManager that knows its plot and knows the current roi, could it provide the counts within the roi as a method?

**Comment by vallsv** [comment]
I am not sure i understand what you mean.

A ROI also knows the plot as far as i can remember. So you already could do what ever you want.

I don't think there is anyway a link bettwen the image you care and the ROI.

Also, you maybe could care about the `StatWidget`, which can link, i think, a ROI and a data.

**Comment by kklmn** [comment]
> A ROI also knows the plot as far as i can remember. So you already could do what ever you want.

Yes to both. Do you see a value in a method of ROI that does image counting within itself (i.e. within that ROI)?

**Comment by kklmn** [comment]
Thank you!
